docs/proposals/Drawer/README.md

```diff
@@ -0,0 +1,70 @@
+# Drawer
+
+The `<Drawer>` component will provide a dismissible side panel.
+
+## Design Patterns
+
+`<Drawer>` It will allow the user to view related content to their main task.
+Unlike a `<Modal>`, it will take full advantage of the available viewport
+height. This will make it ideal for long lists and other scrolling content.
+
+Some scenarios for a `<Drawer>` include the following:
```

Comment:
Do you see a situation where you could drill into more detail from a drawer?

One potential example:
![image](https://user-images.githubusercontent.com/131327/74574662-a3ee6680-4f41-11ea-84c2-9ea35fdc6e71.png)

I could also see drawer content being treated a bit like a [UINavigationController](https://developer.apple.com/documentation/uikit/uinavigationcontroller)?
![](https://docs-assets.developer.apple.com/published/83ef757907/navigation_interface_2x_8f059f7f-2e2f-4c86-8468-7402b7b3cfe0.png)

If there will be navigation within a drawer should we consider how that might work here?

Reply:
Hmmm, maybe. But initially we just need it to display a list or other content. I can't imagine needing anything sophisticated like navigation.

Reply:
If we're counting the AF drawer as being part of this component it currently has similar functionality for its settings customization.

Reply:
Woah! I didn't know this existed within the panel.

![image](https://user-images.githubusercontent.com/1479091/74575487-6dfeb180-4f44-11ea-914b-7e49421df83e.png)

Reply:
@chris-at-jobber Hmmm, do you think I need to incorporate this multi panel thing into the official proposal?

Reply:
I think it's a valid use case, even in a non-overlay drawer - what comes to mind for me is giving the SP control over what kinds of insights they want in the insightful dashboard drawer. If there's no current plan to implement something like that, then maybe it's not necessary for the proposal, but if you think there's a strong likelihood that it would be valuable I would include it.